packages/axelar-wasm-std/src/permission_control.rs

```diff
@@ -0,0 +1,283 @@
+use crate::flagset::FlagSet;
+use cosmwasm_std::{Addr, StdResult};
+use cw_storage_plus::Item;
+use flagset::flags;
+use serde::{Deserialize, Serialize};
+
+flags! {
+    #[repr(u8)]
+    #[derive(Serialize, Deserialize)]
+    pub enum Permission: u8 {
+        NoPrivilege = 1, // this specifies that the user MUST NOT have an elevated role
```

Comment:
What's the usecase for this?

Reply:
no use case on it's own, I need it to define the `Any` case